pkg/v1/tkg/tkgctl/create_cluster_test.go

```diff
@@ -645,71 +692,33 @@ var _ = Describe("Unit tests for feature flag (config.FeatureFlagPackageBasedLCM
-				tkgClient.IsPacificManagementClusterReturnsOnCall(0, true, nil)
-				tkgClient.GetCurrentRegionContextReturns(regionContext, nil)
-				tkgClient.IsFeatureActivatedReturns(true)
-				tkgClient.CreateClusterReturnsOnCall(0, false, nil)
-				options.ClusterConfigFile = classInputFileAzure
-				tkgConfigReaderWriter, _ := tkgconfigreaderwriter.NewReaderWriterFromConfigFile(configFilePath, configFilePath)
-				fg := &fakes.FakeFeatureGateHelper{}
-				fg.FeatureActivatedInNamespaceReturns(false, nil)
-				tkgctlClient := &tkgctl{
-					configDir:              testingDir,
-					tkgClient:              tkgClient,
-					kubeconfig:             kubeConfigPath,
-					tkgConfigReaderWriter:  tkgConfigReaderWriter,
-					tkgConfigUpdaterClient: tkgconfigupdater.New(testingDir, nil, tkgConfigReaderWriter),
-					featureGateHelper:      fg,
-				}
-				// feature flag (config.FeatureFlagPackageBasedLCM) activated, its clusterclass input file, but "clusterclass" feature in FeatureGate is disabled, so throws error
-				err := tkgctlClient.CreateCluster(options)
-				expectedErrMsg := "vSphere with Tanzu environment detected, however, the feature 'clusterclass' is not activated in 'vmware-system-capw' namespace "
-				Expect(err.Error()).To(ContainSubstring(expectedErrMsg))
-			})
-
-
-			It("When featuregate api it self throws error", func() {
-				kubeConfigPath := getConfigFilePath()
-				regionContext := region.RegionContext{
-					ContextName:    "queen-anne-context",
-					SourceFilePath: kubeConfigPath,
-				}
-
-				tkgClient = &fakes.Client{}
-				tkgClient.IsPacificManagementClusterReturnsOnCall(0, true, nil)
-				tkgClient.GetCurrentRegionContextReturns(regionContext, nil)
-				tkgClient.IsFeatureActivatedReturns(true)
-				tkgClient.CreateClusterReturnsOnCall(0, false, nil)
-				options.ClusterConfigFile = classInputFileAzure
-				tkgConfigReaderWriter, _ := tkgconfigreaderwriter.NewReaderWriterFromConfigFile(configFilePath, configFilePath)
-				fg := &fakes.FakeFeatureGateHelper{}
-				errorMsg := "error while feature status in featuregate"
-				fg.FeatureActivatedInNamespaceReturns(true, fmt.Errorf(errorMsg))
-				tkgctlClient := &tkgctl{
-					configDir:              testingDir,
-					tkgClient:              tkgClient,
-					kubeconfig:             kubeConfigPath,
-					tkgConfigReaderWriter:  tkgConfigReaderWriter,
-					tkgConfigUpdaterClient: tkgconfigupdater.New(testingDir, nil, tkgConfigReaderWriter),
-					featureGateHelper:      fg,
-				}
-				// feature flag (config.FeatureFlagPackageBasedLCM) activated, its clusterclass config input file, but "clusterclass" feature in FeatureGate is enabled,
-				// but throws error for the FeatureGate api, so we expect error here.
-				err := tkgctlClient.CreateCluster(options)
-				// as FeatureGate api throws error, we expect error.
-				Expect(err.Error()).To(ContainSubstring(errorMsg))
-			})
-		*/
+		It("When Feature constants.CCFeature is disabled in featuregate, return error", func() {
```

Comment:
As mentioned before try rephrasing so the test output reads more naturally, 
something like (It("returns error when xxx is disabled"). Same for a number of these test cases.